plankton/scripts/run.py

```diff
@@ -61,23 +62,23 @@
                     help='Arguments for the adapter.')
 
 
-def run_simulation(argument_list=None):
+def do_run_simulation(argument_list=None):
     arguments = parser.parse_args(argument_list or sys.argv[1:])
 
     if arguments.version:
-        print(__version__)
-        return
+        return __version__
```

Comment:
I'm really not convinced on this.

`plankton.py`, `__main__.py` and `plankton-control.py` are really just stubs that we use as cross-platform symlinks. They shouldn't contain any real functionality.

Both `scripts/run.py` and `scripts/control.py`, on the other hand, are really console applications rather than modules. They define an `argparser` and use it to parse command line arguments, initializing and running the simulation based on that. It makes sense for them to print to the command line.

It doesn't make sense to force them to only return strings, because this limits how and when they can output to the command line: at the end or when something goes wrong. How would we handle a `--verbose` option? Outputting log messages to the command line?

I also don't think writing unit tests for them is necessary / wouldn't really make sense. They're applications, not units.